Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityInputActionsProfile.asset

```diff
@@ -59,3 +59,6 @@ MonoBehaviour:
   - id: 15
     description: Toggle Profiler
     axisConstraint: 0
+  - id: 16
+    description: Tap
```

Comment:
Do we need `Tap` to be distinct from `Select`? `Select` has always been the equivalent of an airtap, whether in HTK or MRTK.

Reply:
I can see having both become confusing ("Do I want my buttons to respond to taps? Select? If I choose tap, do my buttons work on platforms that don't support 'tap'?" )

Reply:
This is a good point and just something I probably didn't think through enough. My immediate feeling is that using Select is more sensible, but I will look into that a bit more.

Reply:
given that the Select action is already used for PointerUp/PointerDown/PointerClicked, I feel satisfied it's the right choice for this

Reply:
I've updated to using select in almost all places (some I felt it still made sense to keep the "tap" name as that's what the gesture recognizer calls it, but it uses the "select" input action and I called the new gesture type "select" as well) - let me know what you think.